docs/developers/android.md

```diff
@@ -0,0 +1,22 @@
+# Android
+
+You can use Android API in Java in order to use Trezor in your Android
+app. Note that the mobile phone needs to support OTG connection and the
+user needs to have an OTG cable to connect Trezor device to the
+phone.
+
+*See more documentation on the [GitHub page][].*
+
+Examples of Android applications that integrate with Trezor using the
+trezor-android API:
+
+-   [Mycelium][]
```

Comment:
i see this syntax `[something][]` throughout, is it intentional? AFAIK you can just remove the trailing `[]`

Reply:
I used pandoc.org/try/ on the smaller pages, and this is a relic of it. The links worked, so I left it.

I'll do some clearing.